### ISSUE TITLE
Ship an RSS reader with a Bluefin news feed ootb

### ISSUE THREAD

**Issue by castrojo** [ublue-os/bluefin#1485]
The motd is great for sysadmins but we need something graphical, which I've always wanted to do but we never got around to it. 

For GNOME I'd love to ship something modern and minimalist. We don't need a full blown old school style tree reader, we just want to show the ublue-news and announcement RSS feeds from discourse. 

https://universal-blue.discourse.group/tag/bluefin-news.rss

I'm currently posting the release notes in bluefin-news but perhaps we can adjust that since we'd want to ship the release feed from github directly.

https://apps.gnome.org/NewsFlash/

^^ this one looks pretty good and is in GNOME Circles already. 

![image](https://github.com/ublue-os/bluefin/assets/1264109/dc172b7a-b7d7-4253-8cce-ad672e8f8441)

And then let's add it to the topleft logomenu. I'm thinking adding horizontal line and then adding it as "News and Updates" or something similar.

**Comment by nicknamenamenick** [comment]
+1

Also for Aurora, there is an official KDE RSS feed application called [Akregator](https://flathub.org/apps/org.kde.akregator). 

I think this should be implemented for Bazzite as well.

**Comment by nicknamenamenick** [comment]
![Akregator](https://github.com/ublue-os/bluefin/assets/121328689/6bdcf068-cf0a-41b3-9370-80e9ec5520fa)
test for bazzite... on a gnome image :D

**Comment by Raindrac** [comment]
I actively use Newsflash on Bluefin and would highly recommend it! It's a great RSS reader, clean yet fully functional.
Honestly, I preferred it over Akregator even when I was running Plasma on Kinoite. Very clean interface.

https://github.com/ublue-os/bluefin/assets/151923221/f86ad7e5-fba4-443f-bd19-8dc11b40f90e

**Comment by Raindrac** [comment]
I actively use Newsflash on Bluefin and would highly recommend it! It's a great RSS reader, clean yet fully functional. Honestly, I preferred it over Akregator even when I was running Plasma on Kinoite. Very clean interface.

**Comment by nicknamenamenick** [comment]
Yeah if this ends up being shipped then I would recommend using Newsflash even on Aurora.

**Comment by RealVishy** [comment]
Could this Kirigami Application be used for Aurora/Bazzite instead of Newsflash?

https://flathub.org/apps/org.kde.alligator

![Screenshot_20240718_2150192](https://github.com/user-attachments/assets/f3095520-d477-4d69-89a4-416c11e5abcd)

**Comment by castrojo** [comment]
libnotify would work for a notification but a reader would keep historical information. If you clear out your notification center the info is gone.

**Comment by RealVishy** [comment]
I'm currently [investigating](https://discuss.kde.org/t/find-alligator-rss-settings-as-file/19352) where the RSS feeds for Alligator are stored. Once i figure that out i can PR to add it to aurora flatpaks.

**Comment by tulilirockz** [comment]
Oh! Now that we split off the Aurora repo from here we can just ship Newsflash directly

**Comment by castrojo** [comment]
Yeah let's see what that looks like in Newsflash before we PR something. I manually added some feeds locally so I can test, here's how it looks:

![Image](https://github.com/user-attachments/assets/3a89ff0c-514a-4ae0-8d12-1f6936cf1aec)

and

![Image](https://github.com/user-attachments/assets/448f1dcc-0355-4e6d-b0ab-612dd27ba28f)

I'm not sure what's happening with the whitespace on that one.

**Comment by castrojo** [comment]
I think the trick for the feed would be to construct it based on the image right? Like if you're on a gts build just show the GTS feed, if you're on stable, only show stable, etc. News will be generic so that should be fine?

**Comment by tulilirockz** [comment]
I just figured out that you can subscribe to feeds on the command line w/ NewsFlash! This shouldnt be too bad to integrate after all then! ![Image](https://github.com/user-attachments/assets/cc19e8a7-a5cb-4315-b5d3-3b98cbc0b8f6)

**Comment by castrojo** [comment]
Ok now with flatpak preinstall this is way easier. I'm attaching the opml file. I think we should just ship it in the path for the flatpak and install it for everyone. Then we just add it to the drop down menu as "News..." or something.

**Comment by joshyorko** [comment]
@castrojo Working on this PR and have a few questions before submitting:

**What I have ready:**
- NewsFlash added to `system-flatpaks.list` with preinstall protection
- OPML file with Bluefin Discourse news + GitHub releases feeds at `/usr/share/ublue-os/newsflash/bluefin-feeds.opml`
- "News..." menu item added to the logo menu

**Clarification needed:**

1. When you said "ship it in the path for the flatpak" - did you mean users would manually import the OPML file via NewsFlash's File > Import? Or were you expecting auto-configuration?

2. NewsFlash's `--subscribe` CLI flag just opens a dialog, it doesn't auto-add feeds. The only way I found to pre-configure feeds is direct SQLite insertion into NewsFlash's database. Is that acceptable, or would you prefer keeping it simple with manual OPML import?

3. You mentioned attaching an OPML file - I don't see the attachment. Could you share it so I can verify the feed URLs match what you intended?
